### PR TITLE
Remove wildcard copies from docker

### PR DESCRIPTION
**Pull request by stlaz**
Alternative to #802

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
doesn't a directory like `bootkube/manifests/manifests` look strange :) ?

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN make build --warn-undefined-variables
 
 FROM registry.svc.ci.openshift.org/ocp/4.4:base
 RUN mkdir -p /usr/share/bootkube/manifests /manifests
-COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/* /usr/share/bootkube/manifests/
+COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/manifests /usr/share/bootkube/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/cluster-kube-apiserver-operator /usr/bin/
 COPY manifests/*.yaml /manifests/
```
Comment (p0lyn0mial) [inline Dockerfile.rhel7]: I think that those two lines could be simplified to just a single command `COPY manifests /manifests`
```
COPY manifests/*.yaml /manifests/
COPY manifests/image-references /manifests/
```

Comment (stlaz) [reply Dockerfile.rhel7]: that's a good observation, we are not storing anything else in there

**Comment by p0lyn0mial** [comment]
/assign @sttts 

it seems like we have competing approaches on how to fix the issue. We need to choose one :)

**Comment by p0lyn0mial** [comment]
/close

in favour of https://github.com/openshift/cluster-kube-apiserver-operator/pull/802

**Pull request closed without merging**